azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -392,19 +380,48 @@ def __get_agent_cgroups(self, agent_slice, cpu_controller_root, memory_controlle
         def enabled(self):
             return self._cgroups_enabled
 
-        def resource_limits_enforced(self):
-            return False
-
         def enable(self):
             if not self._cgroups_supported:
                 raise CGroupsException("Attempted to enable cgroups, but they are not supported on the current platform")
-
             self._cgroups_enabled = True
 
         def disable(self):
             self._cgroups_enabled = False
             CGroupsTelemetry.reset()
 
+        def set_cpu_quota(self, quota):
+            """
+            Sets the agent's CPU quota to the given percentage (100% == 1 CPU).
+
+            This is done using a dropin file in the default dropin directory; any local overrides on the VM will take precedence
+            over this setting.
```

Comment:
Add the return type of the function in the doc? (i.e. True if success else False)

Reply:
will do

Reply:
updated